osu.Framework/Graphics/Containers/CompositeDrawable.cs

```diff
@@ -567,6 +567,12 @@ protected internal virtual void AddInternal(Drawable drawable)
         /// </summary>
         protected internal void AddRangeInternal(IEnumerable<Drawable> range)
         {
+            if (range is IContainerEnumerable<Drawable> container)
```

Comment:
```suggestion
            if (range is IContainerEnumerable<Drawable>)
```

No need to grab the variable for these calls.

Reply:
Would change to:
```csharp
            if (range is IContainerEnumerable<Drawable>)
            {
                throw new InvalidOperationException($"Attempting to add a {typeof(IContainer)} as a range of children to {this}."
                                                    + $"If intentional, consider using the {nameof(IContainerEnumerable<Drawable>.Children)} property instead.");
            }
```

is that fine?